storage/ndb/src/kernel/blocks/dblqh/DblqhMain.cpp

```diff
@@ -17003,9 +17003,21 @@ void Dblqh::scanReleaseLocksLab(Signal *signal,
   Fragrecord::FragStatus fragstatus = fragptr.p->fragStatus;
   ndbrequire(is_scan_ok(scanPtr, fragstatus));
   check_send_scan_hb_rep(signal, scanPtr, regTcPtr);
+  if (scanPtr->readCommitted) {
+    jamDebug();
+    Uint32 release_counter = scanPtr->scanReleaseCounter;
+    Uint32 curr_rows = scanPtr->m_curr_batch_size_rows;
+    Uint32 num_releases = (curr_rows - release_counter) + 1;
+    if (release_counter >= curr_rows) {
+      num_releases = 1;
+    }
+    scanPtr->scanReleaseCounter += (num_releases - 1);
+    scanLockReleasedLab(signal, regTcPtr);
```

Comment:
What would be the difference if we directly set
scanPtr->scanReleaseCounter = scanPtr->m_curr_batch_size_rows
and call scanLockReleasedLab(signal, regTcPtr) inside this if block?
According to the code at the end of Dblqh::scanReleaseLocksLab(), they should be equivalent before calling scanLockReleasedLab().

Reply:
Can you show me where the cases "scanReleaseCounter = rows + 1 and m_curr_batch_rows = rows" are? I didn't find it.